### PR TITLE
[MM-57263] Tweaks to expanded view

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR implements the UX polish tweaks in https://www.figma.com/file/UIfE0vN1ESsXLHZYDP5eqD/Updates-to-Control-Bar-and-Header except the call settings which I am deferring to a separate PR given it's enough work on its own and mostly decoupled from this.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-57263

### CODE REVIEW

File: webapp/src/components/expanded_view/component.tsx [hunk 1]
```diff
@@ -1025,19 +1044,17 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
                                 </Tooltip>
                             }
                         >
-                            <button
-                                className='button-close'
-                                style={this.style.closeViewButton}
+                            <CloseViewButton
+                                className='style--none'
```
Comment (cpoile) [inline webapp/src/components/expanded_view/component.tsx:1048]: Do we need this className?

**Review by cpoile — APPROVED** [review]
Just some non-blocking comments. Thanks Claudio!

File: webapp/src/components/expanded_view/component.tsx (hunk 1 above)
Comment (streamer45) [reply webapp/src/components/expanded_view/component.tsx:1048]: I'd say yes, as with any other button, we don't want the default browser styling. Before we were overriding everything through `.button-close`. This is more consistent now.

File: webapp/src/components/expanded_view/component.tsx [hunk 2]
```diff
@@ -1025,19 +1044,17 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
...
                                 onClick={this.onCloseViewClick}
                             >
                                 <CollapseIcon
                                     style={{
                                         width: '24px',
                                         height: '24px',
```
[the remark this answers is not in the document]
Comment (streamer45) [reply webapp/src/components/expanded_view/component.tsx:1054]: Looks like I got confused once more.

File: webapp/src/components/expanded_view/reaction_button.tsx [hunk 3]
```diff
@@ -207,8 +207,8 @@ export const ReactionButton = forwardRef(({trackEvent, isHandRaised}: Props, ref
                 icon={
                     <SmileyIcon
                         style={{
-                            width: '28px',
-                            height: '28px',
+                            width: '24px',
+                            height: '24px',
```
[the remark this answers is not in the document]
Comment (streamer45) [reply webapp/src/components/expanded_view/reaction_button.tsx:211]: It should be doable with a common class. Or we could go back to compass icons which were controlled by font sizes I believe :exploding_head:

**Review by abhijit-singh — CHANGES_REQUESTED** [review]
Thanks @streamer45! A few small requests:
1.  For the controls at the bottom, as per the design, the icon size should be 20x20px with a padding of 12px on all sides. Currently the icons are 24x24 with a spacing of 10px. 
2. Similarly for the minimize icon on the top right, the icon size should be 20x20px with 10px space on all sides. 
3. For the `leave call` icon button, there seems to be `0.8` opacity applied on the white icon that we can get rid of. 
4. For the threads button, wondering if we can add a border matching the background color around the notification indicator? I think it might not be straightforward though since the color is a mix of a background color and an overlay.  
image

**Comment by streamer45** [comment]
Thanks @abhijit-singh , will look into the above.

**Pull request merged**